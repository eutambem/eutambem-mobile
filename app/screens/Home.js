import React from 'react';
import PropTypes from 'prop-types';

import Container from '../components/common/Container';
import Button from '../components/common/Button';

const Home = props => (
  <Container>
    <Button
      onPress={() => props.navigation.navigate('CreateReport')}
      title="Criar Relato"
    />
  </Container>
);

Home.propTypes = {
  navigation: PropTypes.shape({
    navigate: PropTypes.func,
  }).isRequired,
};

export default Home;
