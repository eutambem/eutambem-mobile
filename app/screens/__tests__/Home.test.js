import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

const props = {
  navigation: {
    navigate: () => {},
  },
};

describe('Home', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Home {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
