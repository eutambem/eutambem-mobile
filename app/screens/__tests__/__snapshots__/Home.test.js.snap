// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<Container
  innerContainerStyle={Object {}}
>
  <Button
    onPress={[Function]}
    title="Criar Relato"
    type="confirm"
  />
</Container>
`;
